misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/MapAssertionsSpec.kt

```diff
@@ -81,14 +89,32 @@ abstract class MapAssertionsSpec(
             { containsKeyWithNullableValueAssertions(this, keyNullableValue("a") { }, arrayOf()) }
         )
     ) {})
+    include(object : SubjectLessSpec<Map<out String, Int>>(describePrefix,
+        keysFeature.forSubjectLess(),
+        keys.forSubjectLess { isEmpty() },
+        valuesFeature.forSubjectLess(),
+        values.forSubjectLess { isEmpty() },
+        getExistingFeature.forSubjectLess("a"),
+        getExisting.forSubjectLess("a") { isGreaterThan(1) }
+    ) {})
+    include(object : SubjectLessSpec<Map<out String?, Int?>>("$describePrefix[nullable] ",
+        getExistingNullableFeature.forSubjectLess("a"),
+        getExistingNullable.forSubjectLess("a") { toBe(null) }
+    ) {})
 
+    val map: Map<out String, Int> = mapOf("a" to 1, "b" to 2)
+    val fluent = expect(map)
+
+    include(object : AssertionCreatorSpec<Map<out String, Int>>(
```

Comment:
please fuse with 78

Reply:
It seems you did not move the following here:
```
    include(object : AssertionCreatorSpec<Map<String?, Int?>>(
        "$describePrefix[nullable] ", mapNullable,
        getExistingNullable.forAssertionCreatorSpec("$toBeDescr: 2", "b") { toBe(null) }
    ) {})
```
please fuse with 85

Reply:
Indeed, missed that one :(